standalone/hosts/ethiopia/production

```diff
@@ -0,0 +1,52 @@
+[all:vars]
+deploy_env=production
+domain_name=api.et.simple.org
+new_relic_app_env=ethiopia
```

Comment:
Well this is a random piece of info to live in here. What is it for, why does it live here?

Reply:
Currently NewRelic is the only place where we need country name to uniquely identify an env. This variable _is_ a bit random because we don't have a separate var to store the country name. The `deploy_env` variable is a bit overloaded right now, it has two usages throughout the codebase:
- As the simple app env (`demo`, `production` etc.)
- As a unique identifier for deploy names (like `bangladesh-demo`, `india-staging`). 

Adding a `simple_app_env` separate from the `deploy_env` will fix this. This is not a big refactor and we **should** come back to this in a separate PR.